Formula/ats2-postiats.rb

```diff
@@ -1,8 +1,8 @@
 class Ats2Postiats < Formula
-  desc "ATS programming language implementation"
+  desc "ATS2: Unleashing the Potentials of Types and Templates"
```

Comment:
I'd call that a subtitle not description. Something around `ATS is a statically typed programming language that unifies implementation with formal specification.` feels better to me.